src/components/ServiceDetails/TrainingModels/CreateModel/Data/Upload/UploadFromLink.js

```diff
@@ -28,11 +28,20 @@ const StyledInputBase = styled(InputBase)(({ theme }) => ({
   fontSize: 14,
 }));
 
-const UploadFromLink = ({ classes }) => {
+const UploadFromLink = ({ classes, trainingDataLink, setTrainingDataLink }) => {
+  const handleTrainingDataLinkBox = event => {
+    setTrainingDataLink(event.target.value);
+  };
+
   return (
     <div className={classes.uploadFromLinkContainer}>
       <Search>
-        <StyledInputBase placeholder="URL: http://www.url.com/file" inputProps={{ "aria-label": "search" }} />
+        <StyledInputBase
+          placeholder="URL: http://www.url.com/file"
+          inputProps={{ "aria-label": "search" }}
```

Comment:
Why there is a hard-coded value in URL ?

Reply:
That's the placeholder string that's why will add it on constant after completing feature